vagrant/provisioning/roles/mariadb/tasks/main.yml

```diff
@@ -164,5 +164,3 @@
   systemd:
     name: mariadb
     state: restarted
```

Comment:
Why do we need to always restart MariaDB, instead of restarting only when its configuration has changed?

Reply:
Sorry, I didn't get a notification... That's because on Docker, for some reason, MariaDB is not running by default, so I thought we could restart it in any case. It doesn't hurt, just costs a few seconds and that will work in all situations.